src/platform_impl/web/event_loop/window_target.rs

```diff
@@ -231,6 +231,62 @@ impl<T> WindowTarget<T> {
             runner.request_redraw(WindowId(id));
         });
 
+        let runner = self.runner.clone();
+        canvas.on_touch_move(move |device_id, location, force| {
+            runner.send_event(Event::WindowEvent {
+                window_id: WindowId(id),
+                event: WindowEvent::Touch(Touch {
+                    id: device_id as u64,
+                    device_id: DeviceId(device::Id(device_id)),
+                    phase: TouchPhase::Moved,
+                    force: Some(force), // Todo
```

Comment:
What's this TODO referring to?

Reply:
Thanks. I think this is misplaced but I  should consider the meaning of it once.

Reply:
I removed this